_posts/2021-10-5-packages-in-roblox.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Packages: What are they?
+author: TheePBHST
+---
+
+As your game continues to develop and get larger in assets, you may start to realize (or not) that you have the same scripts in different assets in your game, and once you begin to lose track of these assets keeping track of them will get very difficult soon.
+
+So, how could you solve this problem? Well thankfully, Roblox has introduced us a feature called, **Packages**. And no, these aren't the packages that make you look nice and strong, but these are the packages that allow you to make commits to your scripts (or even physical objects) and be able to publish to other places, update other packages, and view the changes to these packages.
+
+In this post, I'll be showing you how to use packages, why should you use them, and other cases for them.
+
+*disclaimer: this post was referenced off of the [Roblox Packages – Reusable Game Assets](https://developer.roblox.com/en-us/articles/roblox-packages) article provided by the Roblox Developer Resources site.*
+
+### Setting Up Your Workspace
+1. Open up a new empty **Baseplate**. *(optional: set ClockTime to 0)*
+2. Create three new [`Parts`](https://developer.roblox.com/en-us/api-reference/class/Part)
+    - Name your either of your three parts to **Red**, **Green**, and **Blue**. *(optional: space them out at least 10 studs)*
+    - Change the colors of the parts to either **Really red**,  **Lime green**, or **Really blue**.
+    - Insert a [`SurfaceLight`](https://developer.roblox.com/en-us/api-reference/class/SurfaceLight) into each part and apply these changes to its properties.
+        - Angle: 10
+        - Brightness: 20
+        - Range: 6
+3. Create a [`Script`](https://developer.roblox.com/en-us/api-reference/class/Script) inside of the `Red` part.
+
+### Editing the Example Script
+So in this new script we've created in the `Red` part, we're going to add a simple piece code.
+
+```lua
+local Part = script.Parent
+local Light = Part.SurfaceLight
+
+Part.Color = Light.Color
+```
+*simple, right?*
```

Comment:
Add some new line boundaries around code blocks, this is necessary in order to render it correctly